features/Steps/UtilitySteps.php

```diff
@@ -45,4 +47,29 @@ public function iWaitSeconds(int $seconds): void
     {
         sleep($seconds);
     }
+
+    /**
+     * @Given I count the :type files
+     */
+    public function iCountTheFiles(string $type): void
```

Comment:
This method name is rather vague. This will store the number of files with the given file extension inside the download folder.